boa/src/syntax/ast/op.rs

```diff
@@ -16,21 +16,79 @@ pub trait Operator {
     }
 }
 
-/// A numeric operation between 2 values
+/// Arithmetic operators take numerical values (either literals or variables)
+/// as their operands and return a single numerical value.
+///
+/// More information:
+///     - MDN documentation:
+/// <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic>
```

Comment:
Given that links are too long, maybe we could shorten them:
```suggestion
///     - [MDN documentation][mdn]
/// 
/// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic
```

Also, do we have a spec link for this? I tried to find something suitable, but didn't find anything :/

Reply:
I couldn't find a spec link.